### ISSUE TITLE
issue using v1.0.0

### ISSUE THREAD

**Issue by JDutil** [laserlemon/vestal_versions#22]
I tried updating to v1.0.0, but I keep running into this undefined method 'changes' error anytime I try to run a rake task or load the rails environment.

jdutils-macbook-pro:searchburlington jdutil$ script/generate vestal_versions
/Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/../lib/vestal_versions/version.rb:16:in `undef_method': undefined method`changes' for class `VestalVersions::Version' (NameError)
    from /Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/../lib/vestal_versions/version.rb:16
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in`gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `require'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:156:in`require'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:521:in `new_constants_in'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:156:in`require'
    from /Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/../lib/vestal_versions.rb:24
    from /Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/../lib/vestal_versions.rb:24:in `each'
    from /Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/../lib/vestal_versions.rb:24
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in`gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `require'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:156:in`require'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:521:in `new_constants_in'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/dependencies.rb:156:in`require'
    from /Library/Ruby/Gems/1.8/gems/vestal_versions-1.0.0/rails/init.rb:1:in `evaluate_init_rb'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin.rb:158:in`evaluate_init_rb'
    from /Library/Ruby/Gems/1.8/gems/activesupport-2.3.5/lib/active_support/core_ext/kernel/reporting.rb:11:in `silence_warnings'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin.rb:154:in`evaluate_init_rb'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin.rb:48:in `load'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin/loader.rb:38:in`load_plugins'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin/loader.rb:37:in `each'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/rails/plugin/loader.rb:37:in`load_plugins'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/initializer.rb:369:in `load_plugins'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/initializer.rb:165:in`process'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/initializer.rb:113:in `send'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/initializer.rb:113:in`run'
    from /Users/jdutil/searchburlington/config/environment.rb:11
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in`require'
    from /Library/Ruby/Gems/1.8/gems/rails-2.3.5/lib/commands/generate.rb:1
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:31:in`require'
    from script/generate:3

**Comment by laserlemon** [comment]
Hi Jeff. Thanks for the report. I've been developing vestal_versions locally as a plugin and apparently there's an issue (that I don't fully understand) with using rails/init.rb rather than just init.rb that was causing the lib files to be loaded twice when installed as a gem.

For now, I've moved init.rb back to the root of the vestal_versions and renamed rails_generators back to just generators. It seems to clear up the issue for me. I bumped the version to 1.0.1. Thanks again for bringing it to my attention so soon and let me know if you come across any other craziness while upgrading.
